.github/workflows/publish.yaml

```
@@ -14,6 +14,7 @@ on:
 jobs:
   build_publish:
     uses: ./.github/workflows/_meta.yaml
+    if: ${{ github.repository == 'jellyfin/jellyfin-docs' }}
```

```suggestion
    if: ${{ contains(github.repository_owner, 'jellyfin') }}
```

Just as an alternative option that we could use across all applicable workflow in the org :thinking:

---

This might be easier when we copy-paste workflows across repositories but it's also less strict (like you said it might false-positive on someone with jellyfin in their name, although unlikely).

I don't particularly have a preference for either of the solutions.

---

Since it allows for easier copy-paste (and thereby less copy-pasta-errors) I have been using this in the new workflows of the CI overhaul.
So I tend to favour this less strict approach, as the likelihood of a user forking with `jellyfin` within their name is a risk I am willing to take.